src/browser/BrowserService.cpp

```diff
@@ -459,17 +429,16 @@ void BrowserService::removeSharedEncryptionKeys()
         return;
     }
 
-    Entry* entry = getConfigEntry();
-    if (!entry) {
-        QMessageBox::information(0,
-                                 tr("KeePassXC: Settings not available!"),
+    Database* db = getDatabase();
+    if (!db) {
+        QMessageBox::information(0, tr("KeePassXC: Settings not available!"),
```

Comment:
Technically if you reach here you do not have an active database. I think this warning can be dropped.